reana_workflow_controller/config.py

```diff
@@ -290,6 +290,9 @@ def _parse_interactive_sessions_environments(env_var):
 DASK_ENABLED = strtobool(os.getenv("DASK_ENABLED", "true"))
 """Whether Dask is enabled in the cluster or not"""
 
+DASK_AUTOSCALER_ENABLED = strtobool(os.getenv("DASK_AUTOSCALER_ENABLED", "true"))
```

Comment:
Regarding "true" vs "True", how things work if people use uppercase or lowercase Helm values?

We may want to be ready to support two scenarios, so do some `lower()` gymnastics here, as we do elsewhere, for example:

```python
os.getenv("K8S_USE_SECURITY_CONTEXT", "True").lower() == "true"
```

P.S. We could amend other occurrences of this boolean handling, see:

```console
$ grep -Ei '"(true|false)' */*/config.py
reana-commons/reana_commons/config.py:    os.getenv("K8S_USE_SECURITY_CONTEXT", "True").lower() == "true"
reana-db/reana_db/config.py:    os.getenv("SQLALCHEMY_POOL_PRE_PING", "False").lower() == "true"
reana-db/reana_db/config.py:    os.getenv("REANA_PERIODIC_RESOURCE_QUOTA_UPDATE_POLICY", "false")
reana-job-controller/reana_job_controller/config.py:    strtobool(os.getenv("REANA_DB_CLOSE_POOL_CONNECTIONS", "false"))
reana-server/reana_server/config.py:DASK_ENABLED = strtobool(os.getenv("DASK_ENABLED", "true"))
reana-server/reana_server/config.py:DASK_AUTOSCALER_ENABLED = strtobool(os.getenv("DASK_AUTOSCALER_ENABLED", "true"))
reana-server/reana_server/config.py:    os.getenv("REANA_USER_EMAIL_CONFIRMATION", "true")
reana-workflow-controller/reana_workflow_controller/config.py:    {"name": "WDB_NO_BROWSER_AUTO_OPEN", "value": "True"},
reana-workflow-controller/reana_workflow_controller/config.py:    os.getenv("REANA_OPENSEARCH_ENABLED", "false").lower() == "true"
reana-workflow-controller/reana_workflow_controller/config.py:    os.getenv("REANA_OPENSEARCH_USE_SSL", "false").lower() == "true"
reana-workflow-controller/reana_workflow_controller/config.py:DASK_ENABLED = strtobool(os.getenv("DASK_ENABLED", "true"))
reana-workflow-controller/reana_workflow_controller/config.py:DASK_AUTOSCALER_ENABLED = strtobool(os.getenv("DASK_AUTOSCALER_ENABLED", "true"))
reana-workflow-engine-cwl/reana_workflow_engine_cwl/config.py:MOUNT_CVMFS = os.getenv("REANA_MOUNT_CVMFS", "false")
reana-workflow-engine-cwl/reana_workflow_engine_cwl/config.py:WORKFLOW_KERBEROS = bool(strtobool(os.getenv("REANA_WORKFLOW_KERBEROS", "false")))
reana-workflow-engine-serial/reana_workflow_engine_serial/config.py:MOUNT_CVMFS = os.getenv("REANA_MOUNT_CVMFS", "false")
reana-workflow-engine-serial/reana_workflow_engine_serial/config.py:WORKFLOW_KERBEROS = bool(strtobool(os.getenv("REANA_WORKFLOW_KERBEROS", "false")))
reana-workflow-engine-snakemake/reana_workflow_engine_snakemake/config.py:MOUNT_CVMFS = os.getenv("REANA_MOUNT_CVMFS", "false")
reana-workflow-engine-snakemake/reana_workflow_engine_snakemake/config.py:WORKFLOW_KERBEROS = bool(strtobool(os.getenv("REANA_WORKFLOW_KERBEROS", "false")))
reana-workflow-engine-yadage/reana_workflow_engine_yadage/config.py:MOUNT_CVMFS = os.getenv("REANA_MOUNT_CVMFS", "false")
reana-workflow-engine-yadage/reana_workflow_engine_yadage/config.py:WORKFLOW_KERBEROS = bool(strtobool(os.getenv("REANA_WORKFLOW_KERBEROS", "false")))
```